.storybook/stories/card/card.stories.ts

```diff
@@ -65,8 +66,6 @@ const defaultParameters: Parameters = {
   },
 };
 
-const variants: Parameters[] = buttonTypes.map(buttonType => ({
-  buttonType,
-}));
-
-setupStorybook([], defaultStory, defaultParameters, variants);
+export const Default: StoryObj = {
+  render: cardTemplate,
+};
```

Comment:
Should be named different from 'Default' before we finish all the refactorings during current condition that ignores `--default` stories for visual testing.
```suggestion
export const Card: StoryObj = {
  render: cardTemplate,
};
```

Reply:
updated the story as per above name